tap_trello/schemas/cards.json

```diff
@@ -10,14 +10,95 @@
         "string"
       ]
     },
-    "checkItemStates": {
+    "customFieldItems": {
       "type": [
         "null",
         "array"
       ],
       "items": {
+        "type": [
+          "null",
+          "object"
+        ],
+        "properties": {
+          "id": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "idValue": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "idCustomField": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "idModel": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "modelType": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "value": {
+            "type": [
+              "null",
+              "object"
+            ],
+            "properties": {
+              "checked": {
+                "type": [
+                  "null",
+                  "string"
+                ]
+              },
+              "date": {
+                "type": [
+                  "null",
+                  "string"
+                ]
+              },
+              "number": {
+                "type": [
+                  "null",
+                  "string"
+                ]
+              },
+              "text": {
+                "type": [
+                  "null",
+                  "string"
+                ]
+              },
+              "option": {
+                "type": [
+                  "null",
+                  "string"
+                ]
+              }
+            }
+          }
+        }
       }
     },
+    "checkItemStates": {
+      "type": [
+        "null",
+        "array"
+      ],
+      "items": null
```

Comment:
I don't think this is a valid schema, I guess this is a new field that came through as a result of the new generation? We should see if it's clear what the data type should be for each item.